src/OrchardCore.Modules/OrchardCore.Users/Drivers/UserRoleDisplayDriver.cs

```diff
@@ -140,17 +139,17 @@ public override async Task<IDisplayResult> UpdateAsync(User user, UpdateEditorCo
 
                     foreach (var role in rolesToRemove)
                     {
-                        if (string.Equals(role, AdministratorRole, StringComparison.OrdinalIgnoreCase))
+                        if (string.Equals(role, RoleNames.Administrator, StringComparison.OrdinalIgnoreCase))
                         {
-                            var enabledUsersOfAdminRole = (await _userManager.GetUsersInRoleAsync(AdministratorRole))
+                            var enabledUsersOfAdminRole = (await _userManager.GetUsersInRoleAsync(RoleNames.Administrator))
                                 .Cast<User>()
                                 .Where(user => user.IsEnabled)
                                 .ToList();
 
                             // Make sure we always have at least one enabled administrator account.
                             if (enabledUsersOfAdminRole.Count == 1 && user.UserId == enabledUsersOfAdminRole.First().UserId)
                             {
-                                await _notifier.WarningAsync(H[$"Cannot remove {AdministratorRole} role from the only enabled administrator."]);
+                                await _notifier.WarningAsync(H[$"Cannot remove {RoleNames.Administrator} role from the only enabled administrator."]);
```

Comment:
this should be changed
```suggestion
                                await _notifier.WarningAsync(H["Cannot remove {0} role from the only enabled administrator.", RoleNames.Administrator]);
```